runtime/doc/builtin.txt

```diff
@@ -4327,6 +4329,27 @@ getregion({pos1}, {pos2} [, {opts}])			*getregion()*
 		Can also be used as a |method|: >
 			getpos('.')->getregion(getpos("'a"))
 <
+getregionpos({pos1}, {pos2} [, {opts}])            *getregionpos()*
+		Returns a list of positions describing the buffer text
+		segments bound by {pos1} to {pos2} buffer.
+		See |getregion()| for the arguments and notes.
+		Note: When the selection is blockwise, it returns a pair of
+		positions for every line:
+                    [[{pos_start}, {pos_end}], ...]
```

Comment:
this note is not needed now, right? because the same structure is always returned

Reply:
The same structure is returned.  But it may be hard to understand why the positions are list.

Reply:
Can you say instead:
```
Same as |getregion()|, but returns a list of positions describing the buffer text
segments bound by {pos1} and {pos2}.

Reply:
OK.  I have fixed it.